requirements/prefect.txt

```diff
@@ -0,0 +1,4 @@
+prefect==3.0.2
+psycopg2==2.9.9
+dune_client==1.7.7
+requests==2.32.3
```

Comment:
prefect_github is probably missing somewhere

Reply:
Thanks! I added it